src/components/molecules/TableHeader/components/EditTableTitleModal/EditTableTitleModal.scss

```diff
@@ -1,6 +1,29 @@
-.edit-title-buttons {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-evenly;
-  padding: 10px 0 0 0;
+.edit-table-modal {
```

Comment:
```suggestion
.EditTableTitleModal {
```

Reply:
Should we be using `padding` values from `scss/constants`?

Reply:
I think so +1